staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go

```diff
@@ -74,12 +82,26 @@ func (m *PathRecorderMux) trackCallers(path string) {
 // refreshMuxLocked creates a new mux and must be called while locked.  Otherwise the view of handlers may
 // not be consistent
 func (m *PathRecorderMux) refreshMuxLocked() {
-	mux := http.NewServeMux()
+	newMux := mux.NewRouter().StrictSlash(true)
```

Comment:
given that all the listed API paths (in swagger, etc) do not have trailing slashes, I could see only serving the canonical one and making the other a redirect. would like a second on that behavior though, and worth a release note